src/main/cpp/t06_longest.cpp

```diff
@@ -17,10 +17,32 @@
 
 #include "t06_longest.h"
 #include <iostream>
+#include <string>
 
 
 using namespace std;
 
 int t06_longest() {
-
+    string currentw = "", resw = "";
+    string a;
+    getline(cin, a);
+    for (int i = 0; i <= a.length(); i++)
+    {
+        if (a[i] != ' ' && i != a.length())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [When i==a.size(), a[i] is out of bounds.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594756)